static/bidder-params/bluesea.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Bluesea Adapter Params",
+  "description": "A schema which validates params accepted by the Bluesea adapter",
+
+  "type": "object",
+  "properties": {
+    "pubid": {
+      "type": "string",
+      "description": "the unique partner account id ",
```

Comment:
nit: remove trailing space after id
`the unique partner account id ` -> `the unique partner account id`